.github/workflows/release.yml

```diff
@@ -0,0 +1,24 @@
+name: Release
+on:
+  push:
+    branches:
+      - master
```

Comment:
see above comments
perhaps we should create a _more_ protected `release` branch and publish from it? Making releases separate prs to the release branch.